Say Their Names/Source/Controller/Home/CollectionViewDataSource/PersonCollectionViewDataSource.swift

```diff
@@ -37,7 +37,6 @@ extension PersonCollectionViewDataSource: UICollectionViewDataSource {
     }
     
     func collectionView(_ collectionView: UICollectionView, viewForSupplementaryElementOfKind kind: String, at indexPath: IndexPath) -> UICollectionReusableView {
-        if collectionView.tag == 0 { return UICollectionReusableView() }
         let headerView = collectionView.dequeueReusableSupplementaryView(ofKind: UICollectionView.elementKindSectionHeader, withReuseIdentifier: PersonHeaderCell.headerIdentifier, for: indexPath) as! PersonHeaderCell
```

Comment:
Why are we casting here? This isn't needed unless we are doing things with `PersonHeaderCell`